lemminx-maven/src/test/resources/pom-plugin-configuration-diagnostic-issue48.xml

```diff
@@ -0,0 +1,77 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Rename this file to `pom-plugin-diagnostic-issue48.xml` as it deals with a bug in the goal & configuration diagnostics.